runtime/vara/src/lib.rs

```diff
@@ -433,12 +433,12 @@ impl pallet_gear_staking_rewards::Config for Runtime {
 
 // TODO: review staking parameters - currently copying Kusama
 parameter_types! {
-    // Six sessions in an era (4 hours)
+    // Six sessions in an era (12 hours)
     pub const SessionsPerEra: sp_staking::SessionIndex = 6;
     // 42 eras for unbonding (7 days)
```

Comment:
Should we fix this comment too?

Reply:
@ekovalev

Reply:
`// 42 eras for unbonding (7 days) ` is obviously misleading given 12 hours epoch duration.
I guess @shamilsan is referring to this comment

Reply:
> `// 42 eras for unbonding (7 days) ` is obviously misleading given 12 hours era duration. I guess @shamilsan is referring to this comment

Exactly.